vembrane/ann_types.py

```diff
@@ -43,11 +45,16 @@ def __getitem__(self, spec):
 IntFloatStr = Union[int, float, str]
 
 
-def type_info(value):
+def type_info(value, number="."):
     if value is None:
         return NA
     if isinstance(value, tuple):
-        return InfoTuple(value)
+        if number != "A":
+            return InfoTuple(value)
+        else:
+            if len(value) != 1:
+                raise MoreThanOneAltAllele()
```

Comment:
Should this also be raised if `number == "G" and len(value) != 2`?

Reply:
Only for diploid cases, I guess?

Reply:
Why? Do the programs to split the VCFs only split "A" but not "G"?

Reply:
Hm, hopefully they handle that correctly.
We also have to check `number == "R" and len(value) != 2`, right?

Reply:
Oh, whoops, I meant to write "R" not "G"!

Reply:
Could be less ugly, but the case `number == "R"` is handled now